marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -190,6 +200,14 @@ getOptions = execParser $ info (helper <*> parser) infoMod
       , showDefault
       ]
 
+    contractQueryPortParser = option auto $ mconcat
```

Comment:
I made a comment about this in the dependent PR, that we should update the deployment configurations.